.php-cs-fixer.dist.php

```diff
@@ -41,10 +24,27 @@
         '@PhpCsFixer' => true,
         '@PhpCsFixer:risky' => true,
         'general_phpdoc_annotation_remove' => ['annotations' => ['expectedDeprecation']], // one should use PHPUnit built-in method instead
-        'header_comment' => ['header' => $header],
+        'header_comment' => ['header' => <<<'EOF'
+            This file is part of PHP CS Fixer.
+
+            (c) Fabien Potencier <fabien@symfony.com>
+                Dariusz Rumiński <dariusz.ruminski@gmail.com>
+
+            This source file is subject to the MIT license that is bundled
+            with this source code in the file LICENSE.
+            EOF],
         'modernize_strpos' => true, // needs PHP 8+ or polyfill
         'no_useless_concat_operator' => false, // TODO switch back on when the `src/Console/Application.php` no longer needs the concat
         'numeric_literal_separator' => true,
     ])
-    ->setFinder($finder)
+    ->setFinder(
+        (new Finder())
+            ->ignoreDotFiles(false)
+            ->ignoreVCSIgnored(true)
+            ->exclude(['dev-tools/phpstan', 'tests/Fixtures'])
+            ->in(__DIR__)
+            ->append([
+                __DIR__.'/dev-tools/doc.php',
```

Comment:
It is already covered 🤔, isn't it?

Reply:
Hmmmmmm, I bet at some point it _wasn't_ (I had it locally for a reason), but it seems like it _is_ now 🤷. I'll remove the commit.